src/CodeReview.Orchestrator/Services/ContainerService.cs

```diff
@@ -23,6 +29,21 @@ public class ContainerService : IContainerService
             _nameFactory = nameFactory ?? throw new ArgumentNullException(nameof(nameFactory));
         }
 
+        private async Task ValidateImage(IDockerClient client, string imageName) {
```

Comment:
Could we renamed this method to `ValidateImageAsync`?

Reply:
done